spec/components/candidate_interface/course_choices_review_component_spec.rb

```diff
@@ -320,6 +320,21 @@
     end
   end
 
+  context 'when an offer has been made for a different course' do
+    it 'renders component with the status as offer and offered course details' do
+      applicationc_choice = create(:application_choice,
+                                   status: 'offer',
+                                   course_option: create(:course_option, :full_time),
+                                   offered_course_option: create(:course_option, :part_time, course: create(:course, description: 'PGCE with QTS part time')))
+      application_form = create(:application_form, application_choices: [applicationc_choice])
+
```

Comment:
Consider using `let()` to setup the data required for this example within the example's context, so the focus within it is on what it's testing rather than the data setup.